qiskit/visualization/pulse/matplotlib.py

```diff
@@ -165,7 +165,7 @@ def to_table(self, name: str) -> List[Tuple[int, str, str]]:
         frequencychanges = self.frequencychanges
 
         for key, val in framechanges.items():
-            data_str = 'framechange: %.2f' % val
+            data_str = 'phase: %.2f' % val
```

Comment:
Should we specify `set phase` and `shift phase`? This might require maintaining `self.framechanges` (should be renamed to `shift_phases`) and `self.set_phases` separately

Reply:
Fixed in https://github.com/Qiskit/qiskit-terra/pull/4540/commits/9a53cc8c41aa0be6fe57cb0368c2c30412b334bb. 

<img width="571" alt="image" src="https://user-images.githubusercontent.com/11485594/84079989-75414380-a9a9-11ea-8503-811fe5277b59.png">